src/app/components/observation/observation-image-carousel/observation-image-carousel.component.scss

```
@@ -75,31 +71,32 @@ ion-fab-button::part(native):hover {
   }
 }
 
-.image-description__inner {
-  overflow: auto;
-  max-width: min(800px, 90%);
-  display: flex;
-  flex-direction: column;
-}
+.image-description__content {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 0.5rem;
+  max-width: 70%;
+  min-width: 670px;
 
-.image-description__title {
-  display: flex;
-  justify-content: space-between;
-  gap: 20px;
-  align-items: center;
-  margin-bottom: 10px;
-  span {
-    font-size: 1.5rem;
-    color: var(--ion-text-color-heading);
-    line-height: 100%;
+  ion-chip {
+    width: fit-content;
+    font-size: 1rem;
+    margin: 0;
   }
 }
 
-@media (max-width: 600px) {
-  .image-description__title {
-    flex-direction: column;
-    gap: 5px;
-    align-items: flex-start;
+.image-descritpion__params {
```

En liten skrivefeil her (husk å endre i template og..)

```suggestion
.image-description__params {
```